README.txt

```diff
@@ -0,0 +1,30 @@
+Sprint 2 Mock Trie
```

Comment:
We should not have a `README.md` and a `README.txt`. GitHub will automatically pretty-format (with Markdown) a `README.md` file, so let's use that. Transfer any information that's missing from here into `README.md` and then delete `README.txt`.